data/loan_programs.json

```diff
@@ -0,0 +1,1461 @@
+[
+  {
+    "id": 43,
+    "loan_program_key": "MA-2",
+    "name": "Energy Saver Home Loan Program",
+    "description": "Low interest loans for income-qualified MA residents for energy efficiency improvements that reduce energy use by at least 20%.",
+    "description_langs": {
+      "es": ""
+    },
+    "status": "PUBLISHED",
+    "state": "MA",
+    "is_national": false,
+    "website_url": "https://www.masshousing.com/en/mass-community-climate-bank/energy-saver-home-loan",
+    "financial_authority_id": 31,
+    "financial_authority": {
+      "id": 31,
+      "authority_type": "state",
+      "description": "",
+      "name": "Massachusetts Community Climate Bank",
+      "state": "MA",
+      "city": "",
+      "image_id": 5,
+      "Image": {
+        "src": "https://storage.googleapis.com/rewiring-america-incentives-assets/capital/generic.png",
+        "height": 100,
+        "width": 100
+      },
+      "created_at": "2025-02-13T22:11:17.978Z",
+      "updated_at": "2025-02-13T22:11:17.978Z"
+    },
+    "eligible_project_types": [
+      {
+        "type": "ELECTRICAL_PANEL_WIRING"
+      },
+      {
+        "type": "ENERGY_AUDIT"
+      },
+      {
+        "type": "ENERGY_STAR_APPLIANCES"
+      },
+      {
+        "type": "WEATHERIZATION"
+      },
+      {
+        "type": "WATER_HEATER"
+      },
+      {
+        "type": "SOLAR"
+      },
+      {
+        "type": "HEATING_COOLING"
+      },
+      {
+        "type": "GEOTHERMAL"
+      },
+      {
+        "type": "EV_CHARGER"
+      }
+    ],
+    "metadata": {},
```

Comment:
My fault-- looks like this should be re-run now that I've merged the PR removing metadata totally?